addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -654,6 +654,7 @@ export function getDeepRange(editable, { range, sel, splitText, select, correctT
     // Target the deepest descendant of the range nodes.
     [start, startOffset] = getDeepestPosition(start, startOffset);
     [end, endOffset] = getDeepestPosition(end, endOffset);
+    const isCollapsed = start === end && startOffset === endOffset;
```

Comment:
About the commit message: "Before this commit there *were* 2 errors that the editor *used to make*"